src/resources/postcss/onboarding.pcss

```diff
@@ -0,0 +1,157 @@
+/*
+ * Onboarding CSS (Tours & Hints)
+ * -------------------------------------------------------------
+ * This file includes styles for the Onboarding Tours and Hints.
+ * These styles go on top of the ones from Intro.js
+ */
+
+/* Utilities */
+@import "utilities/_all.pcss";
```

Comment:
I think you'll need to set the variables skeleton and full as dependencies of this pcss file and remove this, as we've moved onto preserving custom properties.

Reply:
Perfecto! good catch! Yeah, seems like that's what we need to do. Worked on this before the split.